src/components/modules/SecondaryTree.vue

```diff
@@ -155,6 +172,9 @@ export default defineComponent({
       children.forEach((child: EntityReferenceNode) => {
         selectedConcept.children.push(this.createTreeNode(child.name, child["@id"], child.type, child.hasChildren));
       });
+      if(this.totalCount >= 20){
```

Comment:
The "page size" (20) is baked in the code in a couple of places, can it be pulled out into a const?